build.xml

```diff
@@ -0,0 +1,43 @@
+<project name="gsafeed" default="build" basedir=".">
+  <description>Feed Files for GSA</description>
+  <tstamp/>
+  <property name="src.dir" location="src"/>
+  <property name="build.dir" location="build"/>
+  <property name="build-src.dir" location="${build.dir}/src"/>
+  <property file="build.properties"/>
+  <property name="compile.java.version" value="7"/>
```

Comment:
xjc has a "target" option that can be set to 2.0. The doc says "This will allow the generated code to run with JAXB 2.0 runtime environment (such as Java SE 6)." If we want to target Java 6, it's probably worth building and running that way now to make sure we don't use some JAXB feature that was added later.

Reply:
Let's go with Java 6 compatibility, with compile.java.version and the JAXB 2.0 flag. PJ would like the sjc command somewhere. I was thinking that it might be mildly useful to be able to regenerate the files for comparison, but if we fix the lint errors in the generated files the diffs will not be very useful.